packages/plugins/users-permissions/server/services/role.js

```diff
@@ -135,11 +135,9 @@ module.exports = ({ strapi }) => ({
       )
     );
 
-    await Promise.all(
```

Comment:
Promise.all creates race condition. Because there is code that needs to increment the permission order by 1 and promise.all messes that up